osgi/jta/pom.xml

```diff
@@ -75,6 +75,12 @@
 		   <artifactId>jboss-transaction-spi</artifactId>
 		   <version>${version.org.jboss.jboss-transaction-spi}</version>
 		   <scope>provided</scope>
+       <exclusions>
+           <exclusion>
+               <groupId>org.jboss.spec.javax.transaction</groupId>
+               <artifactId>jboss-transaction-api_1.1_spec</artifactId>
+           </exclusion>
+         </exclusions>
```

Comment:
I think the SPI version might need an upgrade instead as the SPI should just have been pulling in 1.2 since 7.0.1: https://github.com/jbosstm/jboss-transaction-spi/releases/tag/7.0.1.Final